.github/workflows/lint.yml

```diff
@@ -19,9 +19,12 @@ jobs:
     - name: Update pip
       run: pip install --upgrade pip
     - name: Install black and pylint
-      run: pip install black pylint
+      run: pip install black pylint ruff
```

Comment:
@cqc-melf `ruff` also supports formatting now. It's compatible with `black` and perhaps can obliviate the need to depend on both tools. See https://docs.astral.sh/ruff/formatter/

Reply:
Thank you for bringing this up, I will do a second round of updates to the extensions to make sure we have all on the same level.